.github/requirements/conda-env-macOS-X64

```diff
@@ -0,0 +1,18 @@
+mkl=2021.2.0
+mkl-include=2021.2.0
+numpy=1.18.5
+pyyaml=5.3
+setuptools=46.0.0
+cmake=3.22.1
+cffi=1.15.1
+typing_extensions=4.3.0
+dataclasses=0.8
+pip=22.2.2
+six=1.16.0
+pillow=9.2.0
+libuv=1.40.0
+pkg-config=0.29.2
+wheel=0.37.1
```

Comment:
Yeah, that could work, but we would need to make a change to test-infra setup-miniconda to accept a list of conda env files I think.  To be honest, I have a feeling that the change would be a complicated to implement in GitHub action yml workflow.  We would need setup-miniconda to be a full fledge GHA like setup-python (implemented in typescripts) to handle this nicely.  So this could be a BE task for later

Reply:
A potential way to do this could be to add an extra setup step to _mac-test.yml similar to:

```
      - name: Setup miniconda (x86, py3.8)
        if: ${{ runner.arch == 'X64' }}
        uses: pytorch/test-infra/.github/actions/setup-miniconda@main
        with:
          python-version: 3.8
          environment-file: .github/requirements/conda-env-${{ runner.os }}

      - name: Setup miniconda (arm64, py3.9)
        if: ${{ runner.arch == 'ARM64' }}
        uses: pytorch/test-infra/.github/actions/setup-miniconda@main
        with:
          python-version: 3.9
          environment-file: .github/requirements/conda-env-${{ runner.os }}
```